src/components/forms/RangeInput/RangeInput.tsx

```diff
@@ -17,9 +19,30 @@ export const RangeInput = ({
   inputRef,
   ...inputProps
 }: RangeInputProps & JSX.IntrinsicElements['input']): React.ReactElement => {
-  // Range defaults to min = 0, max = 100, step = 1, and value = (max/2) if not specified.
-
   const classes = classnames('usa-range', className)
+  // input range defaults to min = 0, max = 100, step = 1, and value = (max/2) if not specified.
+  const defaultMin = 0
+  const defaultMax = 100
+  const { min, max, defaultValue } = inputProps
+  const ariaMin = min || defaultMin
+  const ariaMax = max || defaultMax
+  const calculatedDefaultValue =
+    defaultValue ||
+    (ariaMax < ariaMin ? ariaMin : ariaMin + ariaMax - ariaMax / 2)
+  const convertValueType = (
+    value: string | number | readonly string[]
+  ): number | undefined => {
+    if (typeof value === 'number' || typeof value === 'string') {
+      return Number(value)
+    }
+    return undefined
+  }
+  const [ariaValue, setAriaValue] = React.useState<number | undefined>(
+    convertValueType(calculatedDefaultValue)
+  )
+  const onValueChange: React.ChangeEventHandler<HTMLInputElement> = (e) => {
+    setAriaValue(e.target.valueAsNumber)
```

Comment:
I think we should also handle a case where the consumer passes in their own `aria-` attributes as props.  We only want to use our handling as a fallback.

So add conditional here
 ```
if (!ariaValueNow) setAriaValue(e.target.valueAsNumber)
```

Reply:
done